examples/rl/vm_scheduling/env_wrapper.py

```diff
@@ -0,0 +1,192 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import numpy as np
+
+from maro.rl.learning import AbsEnvWrapper, Transition
+from maro.simulator import Env
+from maro.simulator.scenarios.vm_scheduling import AllocateAction, PostponeAction
+
+
+def post_step(env: Env, tracker: dict, transition: Transition):
+    tracker["env_metric"] = env.metrics
+    if "vm_cpu_cores_requirement" not in tracker:
```

Comment:
If `"vm_cpu_cores_requirement"` not in `tracker`, then add `"vm_core_requirement"` into `tracker`as a key: why used two different keys? Is this a typo or a special design?

Reply:
Thanks for spotting this. It's a typo indeed.